main.go

```diff
@@ -197,6 +197,7 @@ var (
 	}
 	slowLinters = []string{"structcheck", "varcheck", "errcheck", "aligncheck", "testify", "test", "interfacer", "unconvert", "deadcode"}
 	sortKeys    = []string{"none", "path", "line", "column", "severity", "message", "linter"}
+	outputKeys  = []string{"console", "json", "checkstyle"}
```

Comment:
I don't think all this complexity is necessary with only two alternate outputs. Just add a `--check-style` flag.